src/components/Search/Search.js

```diff
@@ -128,7 +129,7 @@ export default class Search extends Component {
 
     const searchClasses = classNames({
       [`${prefix}--search`]: true,
-      [`${prefix}--search--lg`]: !small,
+      [`${prefix}--search--xl`]: !small,
```

Comment:
Do we need to guard these changes with a feature flag? Wasn't sure i this would break for people currently using this, or not.

Reply:
Oh yes good catch